### PR TITLE
Update README with current architecture

### PR DESCRIPTION
**Pull request by ifireball**
Updated the README file to include details about our current shell-script based architecture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,43 +3,47 @@
...
 
 ```mermaid
-flowchart TD
+flowchart TB
     subgraph A["RHTAP clusters"]
         A1[(API server)]
-        A2(["OpenShift
-         logging collector"])
+        A2(["`OpenShift
+        logging collector`"])
```
Comment (yftacherzog) [inline README.md:10]: What's the meaning/purpose of adding the ` accent?

File: README.md [hunk 2]
```diff
@@ -3,43 +3,47 @@
...
+        B3--"Segment events"--> B4
     end
 
-    C3--"User resource
-     actions"-->B2
+    C-- "User resource
+     actions" -->B2
 
     G([Segment])
     H[(Aplitude)]
```
Comment (yftacherzog) [inline README.md:43]: is that a typo? Can we fix it while we're at it? 
```suggestion
    H[(Amplitude)]
```

File: README.md (hunk 1 above)
Comment (ifireball) [reply README.md]: It makes the text be processed as markdown, but unfortunately this is not supported by the Mermaid version used by GitHub, so removed.

File: README.md (hunk 2 above)
Comment (ifireball) [reply README.md]: fixed.

**Review by yftacherzog — APPROVED** [review]

**Review by Omeramsc — APPROVED** [review]
LGTM

**Pull request merged**